apis/core/bootstrap/shapes/dataset.ts

```diff
@@ -189,19 +189,25 @@ ${shapeId} {
     ${sh.property} [
       ${sh.name} "Creation Date" ;
       ${sh.path} ${dcterms.issued} ;
-      ${sh.minCount} 1 ;
       ${sh.maxCount} 1 ;
       ${sh.datatype} ${xsd.date} ;
       ${sh.order} 150 ;
     ] ;
     ${sh.property} [
-      ${sh.name} "Last Update" ;
-      ${sh.path} ${dcterms.modified} ;
-      ${sh.minCount} 1 ;
+      ${sh.name} "Date published" ;
+      ${sh.path} ${schema.datePublished} ;
       ${sh.maxCount} 1 ;
       ${sh.datatype} ${xsd.date} ;
       ${sh.order} 160 ;
     ] ;
+    ${sh.property} [
+      ${sh.name} "Creation Date" ;
+      ${sh.path} ${schema.dateCreated} ;
+      ${sh.maxCount} 1 ;
+      ${sh.datatype} ${xsd.date} ;
+      ${dash.hidden} true ;
+      ${sh.equals} ${dcterms.issued} ;
+    ] ;
```

Comment:
I added a feature to shaperone which automatically keeps properties in sync when one has `dash:hidden true` and `sh:equals`

By restricting this behavior only to `dash:hidden` it will not break "repeat password" kind of forms